pkg/config/ext_authz.go

```diff
@@ -53,11 +53,32 @@ type ExternalAuthzConfig struct {
 	HTTPFilter *hcm.HttpFilter
 }
 
+type extAuthzProtocol string
+
+const (
+	grpc  extAuthzProtocol = "grpc"
```

Comment:
It is better to use `extAuthzProtocolGRPC` or `GRPCextAuthzProtocol` for the name?